### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by zolkis**
Editorial changes + removed the inConference attribute.

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Seems your working directory might be out of date.

File: index.html [hunk 1]
```diff
@@ -973,14 +980,11 @@
...
-          ConferenceCall join(<!--optional DOMString remoteParty-->)
+          ConferenceCall join()
         </dt>
         <dd>
           Initiates merging the active and held calls into a multiparty call. 
-          <!--When called with the <a>remote party id</a> of the new
-    call participant, puts the current call on hold, dials a new call to the
-    participant, then creates a conference call by merging the current call and
-    the new call.--> Using this method MUST be the only way to create a <code>
+          Using this method MUST be the only way to create a <code>
```
Comment (marcoscaceres) [inline index.html:987]: This is a "statement of fact" and not a conformance requirement (i.e., can't be tested), hence it should be:

> Using this method is be the only way to create a ...

File: index.html [hunk 2]
```diff
@@ -1496,22 +1500,22 @@
...
+        </figure></p>
+        <p><figure>
+          <img src="images/outbound_call_state_diagram.gif" alt=
           "Inbound Call State Diagram" title="Inbound Call State Diagram">
           <figcaption>
             The figure depicts the most usual state transitions for dialed
             calls.
           </figcaption>
-        </figure>
+        </figure></p>
```
Comment (marcoscaceres) [inline index.html:1518]: Wrapping block level elements inside a  `<p>` will result in invalid HTML. `<p>` can only contain text and inline elements (like span, strong, etc.).

**Comment by zolkis** [comment]
did a manual merge, over a conflict because a previous patch

**Comment by marcoscaceres** [comment]
I fixed the markup on a local branch.

**Pull request merged**